### PR TITLE
Remove experimental feature warning for story validation and entity roles and groups

### PR DESCRIPTION
**Pull request by aeshky**
**Proposed changes**:

We are moving two features from experimental to official features. This pull request updates the **rasa docs** and the **code**. 

1. **Entity roles and groups** (see: #8791). I removed the experimental feature warning from:
    - The rasa docs
    - The code, where it only appears as a print statement
   
2. **Story validation tool** (see #8024). I removed the experimental feature warning from:
   - The rasa docs only (I could not find a warning in the code)
   
**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by koernerfelicia** [comment]
Looks good to me 🚀  could you add a changelog item?

File: changelog/8791.doc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The experimental feature warning for entity roles and groups has been removed from the rasa docs, and from the code where it used to appear as a print statement.
```
Comment (wochinge) [inline changelog/8791.doc.md:1]: not only from the docs but also when using Rasa Open Source, right?

**Review by wochinge — COMMENTED** [review]
Out of curiosity: Do we want to do the same for incremental training?

File: changelog/8791.doc.md (hunk 1 above)
Comment (aeshky) [reply changelog/8791.doc.md:1]: Yes, @wochinge. Rasa Open Source is what I mean by:
> and from the code where it used to appear as a print statement.

**Comment by aeshky** [comment]
> Out of curiosity: Do we want to do the same for incremental training?

@wochinge It's not yet on our list, as far as I'm aware. @TyDunn?

**Comment by TyDunn** [comment]
These two were decided before. Later, @JEM-Mosig was driving a discussion in [this table](https://www.notion.so/rasa/Experimental-Features-89be5399e9d24d31903716b72c3c9c80) about also making incremental training generally available, but I don't think we resolved it to the point of creating an issue for someone to do it

File: changelog/8791.doc.md (hunk 1 above)
Comment (wochinge) [reply changelog/8791.doc.md:1]: Oh true 🙈 Makes sense 👍🏻

**Comment by aeshky** [comment]
I have a failed check:
[coverage/coveralls — Coverage decreased (-0.001%) to 87.764%](https://coveralls.io/builds/41653646) 
I don't know if this happened after I added the change logs files or after merging main into the branch. 
Any tips for resolving this @wochinge and @koernerfelicia?

**Comment by koernerfelicia** [comment]
@aeshky that's not a required check, and given that you didn't add any code I think can safely be ignored. I guess we should talk to Mariana or Ella about the tolerance of the coverage checks (maybe this can be adjusted?), seems like this is just noise..

**Review by koernerfelicia — APPROVED** [review]
LGTM 🚀

**Comment by aeshky** [comment]
@wochinge Are you ready to approve this PR? 
Also, I am merging to main. Is this correct :) ? This should go in the 2.8.x release according to [this comment](https://github.com/RasaHQ/rasa/issues/8791#issuecomment-886858060).

**Review by wochinge — APPROVED** [review]
🚀

**Comment by wochinge** [comment]
Re the branch: I'd vote for merging this to `main`. I replied / started a discussion with @TyDunn [here](https://github.com/RasaHQ/rasa/issues/8791#issuecomment-888164653)

**Comment by aeshky** [comment]
@wochinge and @koernerfelicia I changed the base branch from `main` to `2.8.x` and resolved the conflicts in the lock file. But the commit now looks very large. Did I make a mistake?

**Pull request closed without merging**